src/Decorators.ts

```diff
@@ -126,5 +126,16 @@ function transformValidateOrReject<T extends object, V extends object>(cls: Clas
 }
 
 function validationErrorsToMessage(errors: ValidationError[]) {
-  return errors.map((e) => Object.values(e.constraints || []).join(', ')).join('. ') + '.'
+  return errors.map((e) => getConstraints(e).join(', ')).join('. ') + '.'
+}
+
+function getConstraints(obj: ValidationError): string[] {
```

Comment:
I would change the method param to be 'error' insted of 'obj', since it would be more descriptive at a glance.